pkg/resources/instance/datasource.go

```diff
@@ -1,4 +1,4 @@
-package exoscale
+package instance
```

Comment:
I can change it, `compute_instance` is named that way only because of the conflict with deprecated resource (for instance pools there is no conflict so we use `instance_pool` and not `compute_instance_pool`). Deprecated resources will not be migrated to a separate package, personally I'd keep the `instance` package name here for simplicity.

Reply:
yes, makes sense, let's keep `instance`